### ISSUE TITLE
Incorrect value for the content-type for Update list item

### ISSUE THREAD

**Issue by mahajangaurav** [SharePoint/sp-dev-docs#6636]
The value of the Content-Type header in the [Update list item](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-lists-and-list-items-with-rest#update-list-item) example is incorrect. 
It should be: "application/json;odata=verbose"
instead of "application/json"

Sending just application/json results in an error as mentioned in this stack exchange: https://sharepoint.stackexchange.com/questions/239169/the-property-metadata-does-not-exist-on-type-sp-data

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e5a8f160-c2fd-8692-1c3f-2c5c44c7a286
* Version Independent ID: 2d7064f8-409c-4465-3fbb-bd6a4c2806f5
* Content: [Working with lists and list items with REST](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-lists-and-list-items-with-rest#using-etag-values-to-determine-document-and-list-item-versioning)
* Content Source: [docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/sp-add-ins/working-with-lists-and-list-items-with-rest.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Jerry0527** [comment]
Yes, should use contentType: "application/json;odata=verbose", otherwise will return 400 error:

`            var listName = "Employee";
            var listItemId = 8;
            var item = {
                "__metadata": { "type": 'SP.Data.EmployeeListItem' },
                "Title": "value change"
            };

            $.ajax({
                url: _spPageContextInfo.siteAbsoluteUrl + "/_api/web/lists/getbytitle('" + listName + "')/items(" + listItemId + ")",
                type: "POST",
                contentType: "application/json;odata=verbose",
                data: JSON.stringify(item),
                headers: {
                    "Accept": "application/json;odata=verbose",
                    "X-RequestDigest": $("#__REQUESTDIGEST").val(),
                    "X-HTTP-Method": "MERGE",
                    "If-Match": "*"
                },
                success: function (data) {
                    alert('Success');
                    
                },
                error: function (data) {
                    alert("Error");

                }
            });
`

**Comment by mahajangaurav** [comment]
Hi Jerry!
Yes, I am aware of the fix now but the issue was more directed towards fixing the documentation here:  [Update list item](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/working-with-lists-and-list-items-with-rest#update-list-item)  

Thanks!

**Comment by Jerry0527** [comment]
May need moderator to push update in the documentation.

**Comment by andrewconnell** [comment]
Sharing is caring :) we welcome community contributions to the docs ;) Care you submit a PR fixing the issue?

**Comment by mahajangaurav** [comment]
would love to! Thanks @andrewconnell

**Comment by mahajangaurav** [comment]
@andrewconnell PR is submitted and is awaiting a merge :) Thanks!
